nvim/lua/colors.lua

```diff
@@ -33,20 +33,3 @@ vim.cmd('highlight! CmpItemKindText guifg=' .. c.fg)
 
 -- highlight on yank
 vim.cmd('highlight! YankPost guifg=' .. c.black .. ' guibg=' .. c.blue)
-vim.api.nvim_create_autocmd('TextYankPost', {
-  pattern = '*',
-  callback = function()
-    vim.highlight.on_yank({higroup='YankPost', timeout=500})
-  end
-})
-
--- :call SynStack() to get the highlight under the cursor
--- https://stackoverflow.com/questions/9464844/how-to-get-group-name-of-highlighting-under-cursor-in-vim
-vim.api.nvim_exec([[
-function! SynStack()
-  if !exists("*synstack")
-    return
-  endif
-  echo map(synstack(line('.'), col('.')), 'synIDattr(v:val, "name")')
-endfunc
-]], true)
```

Comment:
this function didn't seem to work last time I tried. also most difficult highlights can't be figured out using this.